data/json/terrain.json

```diff
@@ -7777,6 +7803,7 @@
     "type": "terrain",
     "id": "t_card_reader_broken",
     "name": "broken card reader",
+    "description": "This is a smartcard reader, but it doesn't seem to be functioning.",
```

Comment:
```suggestion
    "description": "This is a smartcard reader, but it doesn't seem to be functioning.  Probably because there's no more blinking red LED.",
```